README.md

```diff
@@ -1,113 +1,53 @@
+![Logo](d2logo.png)
 
-## Profiling
+[![Patreon](https://img.shields.io/badge/dynamic/json?color=%23e85b46&label=Support%20us%20on%20Patreon&query=data.attributes.patron_count&suffix=%20patrons&url=https://www.patreon.com/api/campaigns/4762180)](https://www.patreon.com/bePatron?u=37261055)
 
-There are many profiler options to debug performance issues. These can be enabled by suppling the following command-line option and are saved in the `pprof` directory:
+----
+[OpenDiablo2](https://opendiablo2.com/) is an ARPG game engine in the same vein of the 2000's games, and supports playing Diablo 2.
 
-`go run . --profile=cpu`
+The engine is written in Go and is cross platform.
 
-Available profilers:\
-`cpu` `mem` `block` `goroutine` `trace` `thread` `mutex`
+> The project does not ship with the assets or content required to play Diablo 2.
+You must have a legally purchased copy of [Diablo 2](https://us.shop.battle.net/en-us/product/diablo-ii) and its expansion [Lord of Destruction](https://us.shop.battle.net/en-us/product/diablo-ii-lord-of-destruction) installed on your computer in order to run that game on this engine.
+If you have an original copy of the disks, those files should work fine as well.
 
-You can export the profiler output with the following command:\
-`go tool pprof --pdf ./OpenDiablo2 pprof/profiler.pprof > file.pdf`
+If you like to contribute to OpenDiablo2, please be so kind to read our [Contribution Policy](./docs/CONTRIBUTING.md) first.
 
-Ingame you can create a heap dump by pressing `~` and typing `dumpheap`. A heap.pprof is written to the `pprof` directory.
+----
 
-You may need to install [Graphviz](http://www.graphviz.org/download/) in order to convert the profiler output.
+## Documentation
 
-## Debugging
+### For Users
 
-### Layouts
+* Buy the official game from Blizzard
+* Locate the MPQ files
+* Install OpenDiablo2 to your system (Linux/Windows/MacOS)
+* [Run it](./docs/play.md) - How to play the game
 
-Layouts can show their boundaries and other visual debugging information when they render. Set `layoutDebug` to `true` in `d2core/d2gui/layout.go` to enable this behavior.
+### For Developers
 
-![Example layout in debug mode](https://user-images.githubusercontent.com/1004323/85792085-31816480-b733-11ea-867e-291946bfff83.png)
+* [Building](./docs/building.md) - Instructions for building the project
+* [Development](./docs/development.md) - Instructions for developers who want to contribute
+* [Profiling](.docs/profiling.md) - Debug performance issues
+* [Debugging](.docs/debug.md) - Common errors and pitfalls
 
-## Roadmap
+### For Contributors
 
-There is an in-progress [project roadmap](https://docs.google.com/document/d/156sWiuk-XBfomVxZ3MD-ijxnwM1X66KTHo2AcWIy8bE/edit?usp=sharing),
-which will be updated over time with new requirements.
+* Design - High-level overview of the OpenDiablo2 org and its projects
+* [Current Status](./docs/status.md) - What you should focus on
```

Comment:
I think, it  could be placed somewhere on the top

Reply:
done ;)